spec.md

```diff
@@ -1076,6 +1076,36 @@ $instance = new class extends \Foo implements
 };
 ~~~
 
+## 9. Enumerations
+
+Enumerations MUST follow the same guidelines as classes, except where otherwise noted here.
+
+Methods in Enumerations MUST follow the same guidelines as methods in classes.  Non-public methods MUST use `private`
+instead of `protected`.
+
+In the case of a backed enum, there MUST NOT be a space between the enum name and colon, and there MUST be exactly one
+space between the colon and the backing type.
+
+Enumeration cases MUST use CamelCase capitalization.
+
+Constants in Enumerations MAY use either CamelCase or UPPER_CASE capitalization.  CamelCase is RECOMMENDED,
+so that it is consistent with cases.
+
+No more than one case is permitted per line.
+
+```php
```

Comment:
This example should have some kind of lead in like:

> The following is an example of a compliant enum:

Reply:
Probably, but the other examples in the document do not.  We should go through and add consistent language to all of them in a separate task.